packages/fxa-auth-db-mysql/CHANGELOG.md

```diff
@@ -6,7 +12,11 @@ No changes.
 
 ### New features
 
-* add vscode tasks for running tests and debugger (dac5e8b98)
+- add vscode tasks for running tests and debugger (dac5e8b98)
```

Comment:
Also not sure why `-` and `*` differences. Were you using an older script @shane-tomlinson? Either way we can clean this up on master after tagging.